Readme.md

```diff
@@ -249,11 +250,15 @@ import { AdSettings } from 'react-native-fbads';
 
 AdSettings contains global settings for all ad controls.
 
+#### deviceHashedId
```

Comment:
Imo it should just be `deviceHash`, or maybe `currentDeviceHash` to denote that it's hash for current device.